FinanceManager.Account/Repositories/AccountLimitRepository.cs

```diff
@@ -0,0 +1,126 @@
+﻿using FinanceManager.Account.Domain;
+using FinanceManager.Account.Models;
+using FinanceManager.UnitOfWork.Abstructs;
+using Microsoft.EntityFrameworkCore;
+
+namespace FinanceManager.Account.Repositories;
+
+public interface IAccountLimitRepository : IUnitOfWorkRepository
+{
+    Task<AccountLimit[]> GetAsync(AccountLimitSpecification specification);
+    Task<bool> CheckExistAsync(AccountLimitSpecification specification);
+    Task<AccountLimit> CreateAsync(
+        string requestId,
+        Guid accountId,
+        AccountLimitType type,
+        AccountLimitTime time,
+        string? description,
+        decimal limitValue);
+    Task UpdateAsync(AccountLimit entity);
+    Task DeleteAsync(AccountLimit entity);
+    Task BulkDeleteAsync(AccountLimit[] entities);
+}
+
+public class AccountLimitSpecification
```

Comment:
Consider splitting file into multiple with Domain term being a folder (e.g. "Accounts")

Reply:
This way you can keep consistent domain separation as well as well-understood code structure for services, repositories and all other required pieces of functionality.